src/css/profile/mobile/common/onoffswitch.less

```diff
@@ -13,7 +13,7 @@
 
     &-input {
         position: absolute;
-        width: 35 * @px_base;
+        width: 37 * @px_base;
```

Comment:
As I know as, 35 is correct size for the checkbox because it has 1px border.
@TomaszLukawskiSam, @lslachciak,
Isn't it?

Reply:
I'll try to verify it.

Reply:
37 is correct because of `box-sizing: border-box;`

Reply:
Oh, I see. Thank you.